src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs

```diff
@@ -17,6 +18,27 @@ public PlaygroundOptions()
             _path = _defaultPath = new PathString("/playground");
         }
 
+        /// <summary>
+        /// The endpoint of the GraphQL service.
+        /// </summary>
+        public string? GraphQLEndpoint
+        {
+            get => _graphQLEndpoint?.AbsoluteUri;
+            set
+            {
+                try
```

Comment:
What is the original exception? I think the exception that Uri throws might be enough.

Reply:
You get `ArgumentNull` or `UriFormatException` - both quite informative, will update :)